mmedit/models/base_archs/gated_conv_module.py

```diff
@@ -32,11 +33,11 @@ class SimpleGatedConvModule(nn.Module):
     """
 
     def __init__(self,
-                 in_channels,
-                 out_channels,
-                 kernel_size,
-                 feat_act_cfg=dict(type='ELU'),
-                 gate_act_cfg=dict(type='Sigmoid'),
+                 in_channels: int,
+                 out_channels: int,
+                 kernel_size: Union[int, tuple[int]],
```

Comment:
I think we can change the typing to `Union[int, Tuple[int, int]]` to align with [MMCV](https://github.com/open-mmlab/mmcv/blob/2478cb6f840206eb0d78cd27320d9068be483860/mmcv/cnn/bricks/conv_module.py#L75).